guides/common/modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc

```diff
@@ -4,8 +4,8 @@
 Use this procedure to enable or disable repositories on an activation key.
```

Comment:
This sentence is no longer accurate, since these instructions now describe only how to _enable_ repositories, and only cover _custom_ repositories.

I'm thinking maybe just update this to be a bit more broad, since the procedure was already there? Will add more suggestions below to that effect

Reply:
I went ahead changed it to the sentence I had in the draft I sent you involving a SCA user.